Packs/ExpanseV2/Scripts/ExpanseRefreshIssueAssets/ExpanseRefreshIssueAssets.py

```diff
@@ -7,11 +7,33 @@
 
```

Comment:
@iyeshaya Please review and add UT
@tomer-pan FYI

Reply:
@iyeshaya this is not necessary anymore, I'm removing the changes I have made to the script.